README.md

```diff
@@ -14,14 +14,45 @@ An extended integration of [orestbida/cookieconsent](https://github.com/orestbid
 
 ## Table of contents
 
+* [How it works](#how-it-works)
 * [Integration into the GTM](#integration-into-the-gtm)
 * [Configuration](#configuration)
 * [Settings modal trigger](#settings-modal-trigger)
 * [Triggering tags based on the consent](#triggering-tags-based-on-the-consent)
 * [Accessing the wrapper in the JavaScript](#accessing-the-wrapper-in-the-javascript)
+* [How to update already published container](#how-to-update-already-published-containers)
 * [Development](#development)
 * [License](#license)
 
+## How it works
```

Comment:
```suggestion
## How the GTM integration works
```

plus update the TOC

Reply:
Let's move this section before the `"How to update already published container"` section. I would like to include some "How it works" sections later, where will be not just the GTM part.

Reply:
done